packages/api/src/apps/datastore_api/app.ts

```diff
@@ -123,6 +124,8 @@ export default async (env: ProcessEnv) => {
       active: lsps.Table("Active LSP", datastores.lspsActive),
       expired: lsps.Table("Expired LSP", datastores.lspsExpired),
     },
+    appStats: appStats.Table("App Stats", datastores.appStats),
+    loopsLock: false,
```

Comment:
Just want to be sure -- does this imply that we _always_ use serverless and the datastore together? i.e. there's no way to run a looping api that ingests to the datastore?

Reply:
The usage of serverless is not tied to storing the data in Google Datastore